_pages/file-a-complaint.md

```diff
@@ -40,8 +42,7 @@ Follow instructions on the [DOJ site](https://civilrights.justice.gov/report/)
 Fill out and send the paper [ADA Complaint Form](https://www.ada.gov/CRT-ReportPDF-Sep2021.pdf) or a letter containing the same information, to:  
 >U.S. Department of Justice  
 Civil Rights Division  
-950 Pennsylvania Avenue, NW  
-4CON, 9th Floor  
```

Comment:
Did you mean to get rid of this line?

Reply:
Yep! Chinelo pointed out that the two addresses did not match, so this is edited to match what we have in the footer.